src/io.c

```diff
@@ -139,7 +139,6 @@ unsigned short io_exchange_al(unsigned char channel, unsigned short tx_len)
 	return 0;
 }
 
-STATIC_ASSERT(CX_APILEVEL == 9, "bad api level");
```

Comment:
Please, do not remove this line. Just assert on apilevel == 9 || ==10

Reply:
(I know that this assert is on two places. In the main it guards generally against new SDK. Here it should guard against PIN_VERIFIED definition